docs/install/install-kn.md

```diff
@@ -34,3 +50,14 @@ Links to either the nightly container image or the latest stable container image
 
 - [Nightly container image](https://gcr.io/knative-nightly/knative.dev/client/cmd/kn)
 - [Latest release](https://gcr.io/knative-releases/knative.dev/client/cmd/kn)
+
+You can run `kn` from container. For example:
+  ```bash
+  docker run --rm --name <your-container-name> -v "$HOME/.kube/config:$HOME/.kube/config" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
```

Comment:
```suggestion
  docker run --rm -v "$HOME/.kube/config:/root/.kube/config" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
```
The original example won't work as expected if the local $HOME is not the same as the home within the image (very likely) as `$HOME` is resolved when you issue this command. 

Also, you don't have to specify a name.